app/js/components/manageApps/ActionAddonModal.jsx

```diff
@@ -21,7 +21,7 @@ export default class DeleteModal extends Component {
             <ModalTitle><span className="dependency-alert">Attention. Dependent Modules Detected!</span></ModalTitle>
           </ModalHeader>
           <ModalBody>
-            <p>The following modules will be shutdown if you decide to continue to delete <strong>{app.name}</strong>:</p>
+            <p>The following modules will be shutdown if you decide to continue to { (action === "stop") ? "stop" : "delete" } <strong>{app.name}</strong>:</p>
```

Comment:
The AirBnB JavaScript style guide recommends single quotes for strings. You can check out this link: https://github.com/airbnb/javascript#strings

Reply:
@joel-ace I am not so sure they are using the AirBnB style configuration, considering I noticed CB uses 'recommended' while resolving certain issues.  What I would however point out, is that it is best practice to be consistent, if using "double-quotes", everything should be double-quoted, and the same for 'single-quotes'.
Javascript allows for the use of both conventions with strings https://www.javascript.com/learn/javascript/strings